planck-cljs/src/planck/shell.cljs

```diff
@@ -25,7 +26,7 @@
 (gobj/set js/global "do_async_sh_callback" do-callback)
 
 (defn- translate-result [js-res]
-  (let [[exit out err] (js->clj js-res)]
+  (let [[exit out err] (->clj js-res)]
```

Comment:
Actually, I think `js-res` is just a JavaScript array, so `seq` might work here.

Reply:
Even simpler, we can just let restructuring operate on the JavaScript array (under the hoods destructuring involves `nth` with `nil` if an item can't be found.)